.devops/templates/build-test-lint.yml

```diff
@@ -18,8 +18,8 @@ steps:
       # @fluentui/api-docs is used within apps/public-docsite-resources/just.config.ts,
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31949&commitId=0567486b4ce05df9ec749d1a896150d324e2f95b&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|5| Changed|
|Avatar Converged.badgeMask.normal.chromium.png|5| Changed|

</details>


  <details><summary><b>SwatchPicker Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.Shape.default.chromium.png|0| Removed|
|SwatchPicker Converged.default - High Contrast.default.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>